.python-version

```diff
@@ -1 +0,0 @@
-3.6.9
```

Comment:
This file was recently committed and merged to master by @alexharv074. I'm taking it back out, since I don't think it was intended to be committed.

Reply:
this is already done with PR #1146

Reply:
Yep, I'll pull master up through the feature branch and into here to resolve this.